kernel/src/include/dmni.h

```diff
@@ -34,3 +34,7 @@ void dmni_read(void *payload_address, size_t payload_size);
  * @param should_free True if should free the payload after the message is sent
  */
 void dmni_send(packet_t *packet, void *payload, size_t size, bool should_free);
+
+void dmni_send_raw(unsigned *packet, size_t size);
+
+void dmni_drop(unsigned size);
```

Comment:
Please document these 2 functions in the same manner the others are documented in the header.